modules/persistent-kubelet-log-level-configuration.adoc

```diff
@@ -0,0 +1,36 @@
+[id="persistent-kubelet-log-level-configuration_{context}"]
+= Persistent kubelet log level configuration
+
+.Procedure
+
+. Use the following `MachineConfig` for persistent kubelet log level configuration:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+ kind: MachineConfig
+ metadata:
+   labels:
+     machineconfiguration.openshift.io/role: master
+   name: 99-master-kubelet-loglevel
+ spec:
+   config:
+     ignition:
+       config: {}
+       security:
+         tls: {}
+       timeouts: {}
+       version: 2.2.0
+     systemd:
+       units:
+         - name: kubelet.service
+           enable: true
+           dropins:
+             - name: 10-kubelet-loglevel.conf
+               contents: |
+                 [Service]
+                 Environment=
+                 Environment="KUBELET_LOG_LEVEL=2"
```

Comment:
Duplicate Environment parameter

Reply:
After correcting above changes and applying MachineConfig, I see `/etc/systemd/system/kubelet.service.d/20-logging.conf` is not getting overridden. I do see a drop-in file created at `/etc/systemd/system/kubelet.service.d/10-kubelet-loglevel.conf` but checking with `systemctl cat kubelet`, I still see old log level. 
@rphillips could you help check if there is something missing here?